src/Dots.js

```diff
@@ -22,6 +23,11 @@ const Dot = styled.button([], {
   space,
   color
 )
+Dot.propTypes = {
+  m: PropTypes.number,
```

Comment:
All `styled-system` functions have prop types, so this could be:

```
Dot.propTypes = {
  ...space.propTypes,
  ...color.propTypes
}
```

Reply:
Should we only use the space and color propTypes? Or should we also declare `m` etc.

Reply:
Those both include all prop types, the space propTypes are here: https://github.com/jxnblk/styled-system/blob/master/src/space.js#L102-L117